packages/atdgen-runtime/atdgen-runtime.2.0.0/opam

```diff
@@ -18,9 +18,9 @@ build: [
 
 depends: [
   "ocaml" {>= "4.02.3"}
-  "jbuilder" {>= "1.0+beta7"}
+  "jbuilder" {>= "1.0+beta9"}
```

Comment:
```suggestion
  "jbuilder" {>= "1.0+beta17"}
```
According to the [changelog](https://github.com/ocaml/dune/blob/main/CHANGES.md#10beta17-01022018=) `diff` was added in beta17.